keydata.go

```diff
@@ -446,6 +446,28 @@ func (d *KeyData) AuthMode() (out AuthMode) {
 	return out
 }
 
+// UnmarshalPlatformHandle unmarshals the JSON platform handle payload to the
```

Comment:
s/to/into/

this should also clarify that handle needs to be a non-nil pointer (that's a requirement from Unmarshal itself but it's better to be explicit).